jekyll/_includes/snippets/secrets-masking.md

```diff
@@ -15,3 +12,6 @@ The value of the environment variable or context will _not_ be masked in the job
 
 Secrets masking will only prevent values from appearing in your job output. Invoking a bash shell with the `-x` or `-o xtrace` options may inadvertently log unmasked secrets (please refer to [Using shell scripts]({{site.baseurl}}/using-shell-scripts)). If your secrets appear elsewhere, such as test results or artifacts, they will not be masked. Additionally, values are still accessible to users [debugging builds with SSH]({{site.baseurl}}/ssh-access-jobs).
 {: class="alert alert-warning"}
+
+The secrets masking feature exists to catch unintentional display of secrets at the output. Best practice is to avoid printing secrets to the output.
```

Comment:
```suggestion
The secrets masking feature exists as a preventative measure to catch unintentional display of secrets at the output. Best practice is to avoid printing secrets to the output. The are many ways that secrets masking could be bypassed, either accidentally or maliciously. For example, any process that reformats the output of a command or script could remove secrets masking.
```